virttest/qemu_monitor.py

```diff
@@ -257,7 +257,9 @@ def __getinitargs__(self):
         # dependency by creating fake VM object which only contains `vm.name`,
         # which is in reality the only information required by Monitor object
         # at this time.
-        return VM(self.vm.name), self.name, self.filename, False
+        # Also ignore errors on unpickle, because failures would lead to
+        # env not being unpickable.
```

Comment:
I believe there is a mistake, `Would lead to env being unpickable` seems to be the intended wording.

Reply:
I'm not native, so maybe it's czechism, but I'm trying to say, that unpickle would not work (pickle works well). So perhaps it's not the best wording, but it's accurate.

Reply:
Ah, OK, so it's a double negative. In this case, let's reduce it to the single positive "failures would lead to env being pickable (possibly add why that is bad)"

Reply:
Well the thing is it's not negative, it's different operation. It is pickable just fine, but during unpickling it might fail and cause the unpickle to fail as whole.

Basically using an analogy with zip and unzip.. `because failures would lead to env not being unzippable`.

Anyway as the wording is hard to make, how about `Always ignore errors during unpickle as exceptions during "__init__" would cause the whole unpickle operation to fail, leaving us without any representation whatsoever.`

Reply:
Make sense. BTW, the code looks good to me.